src/Symfony/Component/DependencyInjection/Tests/Compiler/RegisterAutoconfigureAttributesPassTest.php

```diff
@@ -47,6 +48,7 @@ public function testProcess()
             ->addTag('another_tag', ['attr' => 234])
             ->addMethodCall('setBar', [2, 3])
             ->setBindings(['$bar' => $argument])
+            ->setConstructor('create')
```

Comment:
why this change?

Reply:
This change (now updated with `setFactory`) ensures that the `constructor` option in `#[Autoconfigure]` of `AutoconfigureAttributed` is taken into account by the pass.

Also, I updated everything else according to your feedback 🙏